content/post/es6-destructuring.md

```diff
@@ -19,7 +19,7 @@ We'll look at this in two parts, **array** and **object** destructuring in progr
 
 * **Array destructuring :boom:**
 
-> Let's say you have an array of of my profile.
```

Comment:
Maybe we can remove the `my` too?

Reply:
Yea, I think it makes sense to remove it. Since this isn't even my real profile.